cfme/utils/net.py

```diff
@@ -67,15 +67,15 @@ def net_check(port, addr=None, force=False):
         # First try DNS resolution
         try:
             addr = socket.gethostbyname(addr)
-
-            # Then try to connect to the port
+        except:
+            _ports[addr][port] = False
+        finally:
+            # Try to connect to the port
```

Comment:
Hi @mshriver, using `cfme.utils.ssh` now works, but I had to change order of checks here. The reason is that even though `socket.gethostbyname` won't work with IPv6, `socket.create_connection` can still be used. I believe that this change should make `net_check` still work as originally intented. But I would be dishonest if I said that I truly understand everything that's going on in the fucntion.